src/WebUI/Pods/PodOverview.tsx

```diff
@@ -146,26 +138,21 @@ export class PodOverview extends BaseComponent<IPodOverviewProps> {
             imageDetailsUnavailableTooltipText = localeFormat("{0} | {1}", valueTooltipText || value, Resources.ImageDetailsUnavailableText);
         }
 
-        const itemToRender = hasImageDetails ?
+        return hasImageDetails ?
             <Tooltip overflowOnly>
-                <Link
-                    className="fontSizeM font-size-m text-ellipsis bolt-table-link"
-                    rel={"noopener noreferrer"}
-                    excludeTabStop
-                    onClick={(e) => {
-                        e.preventDefault();
-                        showImageDetails(imageId);
-                    }}
-                >
-                    {value}
-                </Link>
+                <div className="pod-image-link details-card-value-field-size">
```

Comment:
could not be able to use keybaord to reach to this link and also the border of the link is not shown when used with keyboard